samples/net/zperf/src/zperf_udp_receiver.c

```diff
@@ -323,14 +326,29 @@ void zperf_udp_receiver_init(const struct shell *shell, int port)
 			return;
 		}
 
-		ret = zperf_get_ipv4_addr(shell, MY_IP4ADDR,
-					  &in4_addr_my->sin_addr);
-		if (ret < 0) {
-			shell_fprintf(shell, SHELL_WARNING,
-				      "Unable to set IPv4\n");
-			return;
+		if (MY_IP4ADDR) {
```

Comment:
and here

Reply:
> and here

Same reason becase of compile error if MY_IP6ADDR is NULL.